<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetWidget</class>
 <widget class="QWidget" name="ResetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>1500</width>
     <height>920</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{background: #FFFFFF;
border-radius: 8px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="PictureTipsWidget" name="video_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>1300</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_Btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>850</y>
      <width>1501</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: #0D9DDB;border-radius: 8px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 8px;color:#333333;}</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1501</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{background: #0D9DDB;border-top-left-radius: 8px;border-top-right-radius:8px}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>提示</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="close_Btn">
     <property name="geometry">
      <rect>
       <x>1410</x>
       <y>20</y>
       <width>44</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/close.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="deviceResetTip_GroupBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>611</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#deviceResetTip_GroupBox{border:none;}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="deviceResetTip_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>571</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#0D9DDB;</string>
     </property>
     <property name="text">
      <string>设备复位中请稍后...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>521</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#progressBar{border:2px solid #0D9DDB;border-radius:5px;}
#progressBar::chunk{background-color:#49B9EA;width:10px;margin:2px;}
</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureTipsWidget</class>
   <extends>QWidget</extends>
   <header>picturetipswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
